common/persistence/sql/sqlplugin/history_tree.go

```diff
@@ -56,10 +56,26 @@ type (
 		BranchID primitives.UUID
 	}
 
-	// HistoryNode is the SQL persistence interface for history trees
+	HistoryTreePrimaryKey struct {
+		ShardID  int32
+		TreeID   primitives.UUID
+		BranchID primitives.UUID
+	}
+
+	// HistoryTreeBranchPage is a struct which represents a page of history tree branches to query.
+	HistoryTreeBranchPage struct {
+		ShardID      int32
+		TreeID       primitives.UUID
+		BranchID     primitives.UUID
+		LatestBranch *HistoryTreePrimaryKey
```

Comment:
what's this for?

Reply:
Ah, that's an artifact of an earlier revision